CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ Full documentation for rocDecode is available at [https://rocm.docs.amd.com/proj
 * Clang is now the default CXX compiler.
 * The new minimum supported version of va-api is 1.16.
 * New build and runtime options have been added to the `rocDecode-setup.py` setup script.
+* Added FFMpeg based software decoding into utils.
+* Modified videodecode sample to allow FFMpeg based decoding
```

Comment:
Do these two lines need to be in the 6.3 release notes?

Reply:
@spolifroni-amd: That will go to 6.4 I think. It doesn't need to be on 6.3 release notes